iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -1121,6 +1129,18 @@ def type(self) -> "Register":
         return Register[(self.dim, self.dtype)]
 
 
+@define_op("set_prio")
+@dataclass
+class SetPrio(CustomOp):
```

Comment:
Can I suggest you rename this to SetWavePriority? We dont have to stick with the instruction name and this really clarifies what is happening.

Reply:
makes sense, done!